cmd/cluster/create.go

```diff
@@ -272,3 +291,59 @@ func CreateCluster(ctx context.Context, opts Options) error {
 
 	return nil
 }
+
+func createInfraNone(ctx context.Context, opts Options, pullSecret []byte, sshKey []byte, client crclient.Client, annotations map[string]string) error {
```

Comment:
It seems that the only need for this is to look up the `master_0.Address`.  Could we take that on the CLI and have no `infra` stage for platform `None`?

Reply:
By now, I think you're right, but the idea is prepare the function to deploy infra for baremetal when CAPI will be implemented.

Reply:
My idea is basically prepare for the next contribution to CAPI for baremetal. This is why I prefer create the new function to allocate the infra creation.

Reply:
Changed with options from flag instead of looking for the address